roles/DCOS.agent_windows/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+- name: Windows | Defining DC/OS by using 'version' or, more specifically, 'image_commit'
+  set_fact:
+     dcos_version_specifier: "{{ dcos_version | default('1.13') }}"
+     ansible_password: '{{ hostvars[inventory_hostname].pass }}'
+
+
+- debug:
+    msg: "dcos_version_specifier = {{ dcos_version_specifier }} , pass = {{ ansible_password }}, bucket_name = {{ bucket_name }}"
+
+
+#- name: Windows | edit Powershell Start script with local IP address of Mesos Master
+#  replace:#
+#    path: /files/start.ps1
+#    regexp: 'local_ip_master'
+#    replace: '{{  hostvars[groups.masters[0]].inventory_hostname }}'
+#  delegate_to: localhost
+
+
+- name: Windows | create folder for temporaly files
+  file:
+    path: /files
```

Comment:
at least this should be specific to DC/OS e.g. `/dcos`

Reply:
removed this step as it will be created recursively in next step